src/App.css

```diff
@@ -1,65 +1,305 @@
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.book-title {
+  width: 12.313rem;
+  height: 1.813rem;
+  font-family: RobotoSlab, Arial, Helvetica, sans-serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
+  margin: 0 0;
+}
+
+.book-author {
+  width: 6.688rem;
+  height: 1.188rem;
+  margin: 0 0;
+  font-family: RobotoSlab, Arial, Helvetica, sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+}
+
+.buttons-div {
+  padding-top: 1rem;
+  display: flex;
+  gap: 0.8rem;
+}
+
+.button {
+  height: 1.188rem;
   background-color: transparent;
-  border: none;
-  color: rgb(128, 17, 17);
+  font-family: RobotoSlab, Arial, Helvetica, sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  border: unset;
+  padding: 0;
   cursor: pointer;
 }
 
+.button-divider {
+  color: #e8e8e8;
+}
+
+.progress-details {
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+}
+
+.progress-image {
+  width: 50px;
+  height: 50px;
+}
+
+.percentage-div {
+  display: flex;
+  flex-direction: column;
+}
+
+.percentage {
+  width: 4.25rem;
+  height: 2.438rem;
+  font-family: Montserrat, Arial, Helvetica, sans-serif;
+  font-size: 2rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.progress-divider {
+  width: 1rem;
+  border-right: 1px solid #8b4747;
+  background-color: #0290ff;
+}
+
+.completed-text {
+  width: 5rem;
+  height: 1.125rem;
+  opacity: 0.5;
+  font-family: Montserrat, Arial, Helvetica, sans-serif;
+  font-size: 0.875rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.chapter-details {
+  display: flex;
+  flex-direction: column;
+}
+
+.current-chapter {
+  width: 7.875rem;
+  height: 1.125rem;
+  opacity: 0.5;
+  font-family: RobotoSlab, Arial, Helvetica, sans-serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.chapter-number {
+  width: 4.688rem;
+  height: 1.313rem;
+  font-family: RobotoSlab, Arial, Helvetica, sans-serif;
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+  color: #121212;
+}
+
+.update-button {
+  width: 10rem;
+  background-color: #0290ff;
+  border: unset;
+  font-family: RobotoSlab, Arial, Helvetica, sans-serif;
```

Comment:
- [ ] Excellent🎉😍 work so far in aligning with the project requirements and the Zeplin design! 
- [ ] I would like to kindly 🙏🙏request some adjustments for further refinement. Specifically, the styling of the `Update Progress` button and the presentation of the `Current Chapter` section require some attention.🤔🤔

- [ ] Currently, the "Update Progress" button appears compressed and does not align with the suggested design. It would greatly enhance the overall aesthetics if you could adjust its width to ensure it expands more in line with the design guidelines.🥱👌

- [ ] Additionally, there seems to be a vertical divider missing between the `progress bar `and the `Update Progress` button, which is an essential element in our design. Integrating this divider will contribute to a more polished and visually appealing design.

- [ ] By implementing these refinements, your design will truly stand out and closely align with our envisioned aesthetic. Your continued dedication to detail and precision is greatly appreciated!🎉®️✔️

| Your Design | Zeplin Design |
|--------|--------|
| ![your](https://github.com/firepicaso/bookstore_react/assets/118761303/b3535dad-2c35-49f8-b475-d361b428cf39) | ![zeplin](https://github.com/firepicaso/bookstore_react/assets/118761303/be95f963-ec17-40fd-a178-41e4f129a012) |